pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -560,32 +560,31 @@ def avg_time(cimtime, num_ops):
             number_of_operations = inst['NumberOfOperations']
 
             cimom_avg_elapsesd_time = avg_time(inst['CimomElapsedTime'],
-                                               number_of_operations)
+                                               number_of_operations) / 1000
 
             provider_avg_elapsesd_time = avg_time(inst['ProviderElapsedTime'],
-                                                  number_of_operations)
+                                                  number_of_operations) / 1000
 
             avg_request_size = avg_value(inst['RequestSize'],
                                          number_of_operations)
             avg_response_size = avg_value(inst['ResponseSize'],
                                           number_of_operations)
 
-            rows.append([number_of_operations,
-                         "{:.1f}".format(cimom_avg_elapsesd_time),
-                         "{:.1f}".format(provider_avg_elapsesd_time),
-                         "{:.1f}".format(avg_request_size),
-                         "{:.1f}".format(avg_response_size),
-                         op_type_name])
+            rows.append([op_type_name,
+                         number_of_operations,
+                         "{:.3f}".format(cimom_avg_elapsesd_time),
+                         "{:.3f}".format(provider_avg_elapsesd_time),
+                         "{:.0f}".format(avg_request_size),
+                         "{:.0f}".format(avg_response_size)])
 
-        headers = ['Operation\nCount', 'Server\ntime(usec)',
-                   'Provider\nTime(usec)', 'Request\nsize(bytes)',
-                   'Response\nsize(bytes)',
-                   'Operation Name']
+        headers = ['Operation', 'Count', 'Server Time\n[ms]',
```

Comment:
Would it be logical to pack up the headers more and into multiple lines so that the column header widths do not greatly exceed the expected column data widths making the table fit into a more narrow terminal width.  Ex. 'Server\nTime [ms]`, 'Request\nsize[b]`

Reply:
There are multiple factors to consider: Readability and width.

With the current implementation, the client statistics is 104 chars wide and the server statistics is 96 chars wide. That is below our standard width of 120 characters.

If we optimize the column headers for with as you suggest, we get down to 91 chars for the client statistics and to 81 chars for the server statistics. That is still above the 80 characters I suppose you try reaching.

So I would go for the better readability.
**DISCUSSION**

Reply:
I am tending to agree that our goal should probably be 80 for very simple tables that make sense in 80 but 120 in general for tables that do not make sense for 80 columns.  You convinced me.  I was just too lazy to expand my standard terminal width to 120 columns.

geneally keeping these down to 120 columns would be logical to me.

Reply:
We agreed that 120 is fine for now. Resolved.